test/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:18.04
+
+ENV mirversion="1.5.0"
+
+RUN apt-get update && apt-get -yy install curl openssl
+
+# Install NVM
+SHELL ["/bin/bash", "-c"]
+# nvm environment variables
+ENV NVM_DIR /usr/local/nvm
+ENV NODE_VERSION 16.14.0
+
+# install nvm
+# https://github.com/creationix/nvm#install-script
+RUN curl --silent -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.2/install.sh | bash
+
+# install node and npm
+RUN source $NVM_DIR/nvm.sh \
+    && nvm install $NODE_VERSION \
+    && nvm alias default $NODE_VERSION \
+    && nvm use default
+
+# add node and npm to path so the commands are available
+ENV NODE_PATH $NVM_DIR/v$NODE_VERSION/lib/node_modules
+ENV PATH $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
+
+# confirm installation
+RUN node -v
+RUN npm -v
+SHELL ["/bin/sh", "-c"]
+
+# Download and install the full service mirror package
+RUN curl -L -o full-service-mirror.tar.gz https://github.com/mobilecoinofficial/full-service-mirror/releases/download/v$mirversion/linux-v$mirversion-testnet.tar.gz
```

Comment:
It would be good if we had a way to test a release before we publish it. Any thoughts on how we can accomplish that?

Reply:
I think the easiest way to do this is to just call the create release scripts, copy it to the appropriate location, then call the run.sh. It sounds like, from discussion with Marco, in the future it'll be able to run just anytime you create dev branch and push to it if we want. But I'm not sure if there's a clean way to incorporate the build into this docker, so I defer to his expertise on this. It seems like this pr doesn't interfere with the CI integration he is doing and will be useful as is so I think it should go in without the additional build logic which will be dealt with elsewhere.